termd-build-driver/src/test/java/org/jboss/pnc/termdbuilddriver/TermdBuildDriverTest.java

```diff
@@ -151,4 +157,45 @@ public void shouldSetLogsInBuildExecution() throws Exception {
 //        verify(buildExecutionMock, times(1)).clearLogsWebSocketLink();
     }
 
+    @Test(timeout = 60_000)
+    public void shouldFetchFromGitAndBuild() throws Exception {
+        //given
+        Path tmpRepo = Files.createTempDirectory("tmpRepo");
+        String repoPath = "file://" + tmpRepo.toAbsolutePath().toString() + "/test-repo";
+        ZipUtils.unzipToDir(tmpRepo, "/repo.zip");
```

Comment:
Do we need to test git at all? It is important that we test communication with BuildAgent. From a BuildDriver perspective it should be the same if you run `git clone` or `whoami`, important is that command is executed and result received.